src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int minfigures = 5;
+    private final double defaultvalue = 10;
+    private final String defaultcolor = "white";
```

Comment:
these should be constants, not variables
private static final and with proper naming